src/BuildScriptGenerator/Node/NodeDependencyChecker.cs

```diff
@@ -19,11 +20,19 @@ public class NpmDependencyChecker : IChecker
             { "eslint-plugin-jsx-ally", "eslint-plugin-jsx-a11y" }
         };
 
+        private readonly ILogger<NodeDependencyChecker> _logger;
+
+        public NodeDependencyChecker(ILogger<NodeDependencyChecker> logger)
+        {
+            _logger = logger;
+        }
+
         public IEnumerable<ICheckerMessage> CheckSourceRepo(ISourceRepo repo)
         {
             dynamic packageJson = NodePlatform.GetPackageJsonObject(repo, null);
             if (packageJson == null)
             {
+                _logger.LogDebug("packageJson is null");
```

Comment:
It would be helpful to have more context on this message as we read through the logs, to provide a semantic value rather than pure state. e.g. "Node checker skipping repo since packageJson is null"

Reply:
Don't forget this message will have a `LoggerName` costume dimension with it - do you think the extra context is still worth adding?

Reply:
Yes. It's easier to see what's going on when reading through the files. Besides, this message depends on the existing version of the code to see what that means. If we change this code at some point, we may not be able to understand the right context in which this was presented.